sources/packages/backend/apps/api/src/route-controllers/education-program/_tests_/e2e/education-program.aest.controller.getProgramsSummary.e2e-spec.ts

```diff
@@ -0,0 +1,775 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitution,
+  createFakeInstitutionLocation,
+  createFakeUser,
+} from "@sims/test-utils";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createFakeEducationProgram,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  Institution,
+  InstitutionLocation,
+  ProgramStatus,
+  User,
+} from "@sims/sims-db";
+import * as request from "supertest";
+import {
+  appendRequiredString,
+  removeUnwantedString,
+} from "@sims/test-utils/utils/string-utils";
+
+describe("EducationProgramAESTController(e2e)-getProgramsSummary", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let institution: Institution;
+  let institutionLocation: InstitutionLocation;
+  let sharedUser: User;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    sharedUser = createFakeUser();
+    await db.user.save(sharedUser);
```

Comment:
Is there a need to create a user to be consumed?
If the methods where it is provided are optional and are not needed for the test, there is no need to create and associate them. 
If the tests require them, please disregard this comment.

Reply:
Yes, the tests need it.